api/webserver.go

```diff
@@ -0,0 +1,16 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/gin-gonic/gin"
+)
+
+func WebServer(port *int) {
```

Comment:
well because in the main.go I'm using a flag which return a pointer for the port.So i used pointer in the func arg so  i don't need to dereference the pointer in the main.but i can dereference in the func calling atg too.what do you reckon?